cirq-core/cirq/ops/control_values.py

```diff
@@ -22,10 +22,21 @@
 
 
 class AbstractControlValues(abc.ABC):
-    """AbstractControlValues is an abstract immutable data class.
-
-    AbstractControlValues defines an API for control values and implements
-    functions common to all implementations (e.g.  comparison).
+    """AbstractControlValues is an abstract class that defines an API for
+
+       control values and implements functions common to all implementations
+       (e.g.  comparison).
+
+    `cirq.ControlledGate` and `cirq.ControlledOperation` are useful to augment
+    existing gates
+    and operations to have one or more control qubits. For every control qubit,
+    the set of
+    integer values for which the control should be enabled is represented by one
+    of the implementations of `cirq.AbstractControlValues`.
+    Implementations of `cirq.AbstractControlValues` can use different internal
+    representations to store control values, but they have to inherits from this
+    abstract class or one of its descendet, satisfy the public API defined here,
+    and be immutable.
```

Comment:
nit: Remove extra whitespace at the end of lines and add line breaks only to separate different paragraphs. 
```suggestion
    `cirq.ControlledGate` and `cirq.ControlledOperation` are useful to augment
    existing gates and operations to have one or more control qubits. For every 
    control qubit, the set of integer values for which the control should be enabled 
    is represented by one of the implementations of `cirq.AbstractControlValues`.
    
    Implementations of `cirq.AbstractControlValues` can use different internal
    representations to store control values, but they must satisfy the public API 
    defined here and be immutable.
```